docs/blog/2020-02-05-accessible-client-side-routing-improvements/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Accessibility Improvements to Client Side Routing - Available in 2.19.8"
+date: 2020-02-05
+author: "Madalyn Parker"
+excerpt: "We recently released accessibility improvements to client side routing, enabling screen reader users to successfully navigate sites built with Gatsby."
+tags: ["accessibility, client-side-routing, diversity-and-inclusion"]
+---
+
+We recently released accessibility improvements to client side routing, building on previous focus management improvements [released in version 2.13.2](https://github.com/gatsbyjs/gatsby/pull/13197). These improvements enable people relying on screen readers to successfully navigate sites built with Gatsby. If you'd like an in-depth look at how we made incremental changes to get to this release, have a look at this [issue](https://github.com/gatsbyjs/gatsby/issues/5581) opened way back in May 2018.
```

Comment:
```suggestion
We recently released accessibility improvements to client side routing, building on previous focus management improvements [released in version 2.13.2](https://github.com/gatsbyjs/gatsby/pull/13197). These improvements enable people relying on screen readers to successfully navigate sites built with Gatsby. If you'd like an in-depth look at how we made incremental changes to get to this release, have a look at this ["page nav doesn't trigger assistive tech" issue](https://github.com/gatsbyjs/gatsby/issues/5581) opened way back in May 2018.
```
I'm not sold on that as the link, but issue felt a bit vague?

Reply:
Yeah I'm not sure how to describe it in a way that's helpful but not overly verbose. 🤔

Reply:
Maybe just issue focused on assistive technology?